spec/src/main/asciidoc/core/definition.asciidoc

```diff
@@ -839,6 +838,28 @@ We may specify that all mock objects are alternatives:
 public @interface Mock {}
 ----
 
+[[priority_stereotype]]
+
+===== Declaring stereotype with `@Priority`
+
+A stereotype may declare a `@Priority` annotation which functions as a means of enabling and ordering affected alternatives, interceptors and decorators.
```

Comment:
Decorators are Full-only, but it probably doesn't make sense to move that single thing out of Lite. Maybe just something like

> affected alternatives, interceptors, and decorators (in {cdi_full})

Note I added an Oxford comma :-) I don't think the spec uses it consistently, but in this case, it makes it visually more clear that "in CDI Full" only applies to decorators.